databroker/headersource/base.py

```diff
@@ -7,10 +7,7 @@
 
 
 class MDSROTemplate(object):
-    def __init__(self, config, version=1, auth=False):
-        if auth:
-            raise NotImplementedError("This implementation of MDS does not "
-                                      "support authentication.")
+    def __init__(self, config, version=1):
```

Comment:
While you're here, do you want to [move version into the config dict](https://github.com/NSLS-II/databroker/issues/185)?